pkg/api/openshiftcluster.go

```diff
@@ -248,6 +251,8 @@ const (
 	FipsValidatedModulesDisabled FipsValidatedModules = "Disabled"
 )
 
+type OIDCIssuer string
```

Comment:
we have doc comments in the admin version, should we also add some doc comments here?